src/service/search/datafusion/mod.rs

```diff
@@ -16,8 +16,8 @@ use crate::meta::functions::ZoFunction;
 
 mod date_format_udf;
 pub mod exec;
-mod match_udf;
-mod regexp_udf;
+pub mod match_udf;
+pub mod regexp_udf;
```

Comment:
```suggestion
pub(crate) mod match_udf;
pub(crate) mod regexp_udf;
```
There should be a reason to use `put`, we shouldn't default to it. Compiler won't report unused symbols if they are `pub`.